python/tester/upload.py

```diff
@@ -305,7 +305,8 @@ def get_samples_lsst(bucket, instrument):
             detector=_DETECTOR_FROM_RS[instrument][m["raft_sensor"]],
             groupId=md["GROUPID"],
             nimages=INSTRUMENTS[instrument].n_snaps,
-            filters=md["FILTBAND"],
+            # ComCam currently sets the FILTBAND header to null.
+            filters=md["FILTBAND"] or md["FILTER"],
```

Comment:
I assume this doesn't cause problems with our existing LATISS data?

Reply:
Since our existing LATISS data have proper `FILTBAND`, the rest is just ignored.